Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -1261,9 +1261,9 @@ You can also use [jadx](0x08-Testing-Tools.md#jadx "jadx") to identify exported
 </manifest>
 ```
 
-The above example from the vulnerable banking application [InsecureBankv2](0x08-Testing-Tools.md#android "Vulnerable applications for Android") shows that only the broadcast receiver named `com.android.insecurebankv2.MyBroadCastReceiver` is exported.
+The above example from the vulnerable banking application [InsecureBankv2](0x08b-Reference-Apps.md#insecurebankv2 "Vulnerable applications for Android") shows that only the broadcast receiver named `com.android.insecurebankv2.MyBroadCastReceiver` is exported.
```

Comment:
Same as in here, could you please include such references for the rest of the apps? 

- For instance, iGoat/iGoat-swift, MSTG Hacking Playground (Android), DVIA/DVIA-v2 are being used but are not yet referenced with `0x08b-Reference-Apps.md#`. there might be more. 
- Some are not used at all and we'd like to know that just by searching for `0x08b-Reference-Apps.md#` in the MSTG. That should get us an overview of where each app is being used.

Reply:
Will search and add the references 🤓

Reply:
Thank you very much!